bin/create_maf_database.py

```diff
@@ -1,24 +1,38 @@
+"""Script to invoke creating maf database"""
 #! /usr/bin/env python3
 import argparse
 import logging
+import time
 
-from genie import input_to_database
 from genie import process_functions
 
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
-
+# This function can further be abstracted once things are further fixed
 def main(db_mapping_id, pemfile, debug):
-
+    """Creates maf database"""
     syn = process_functions.synLogin(pemfile, debug=debug)
 
     query = 'SELECT * FROM {}'.format(db_mapping_id)
-    databaseToSynIdMapping = syn.tableQuery(query)
-    databaseToSynIdMappingDf = databaseToSynIdMapping.asDataFrame()
+    database_mapping = syn.tableQuery(query)
+    database_mappingdf = database_mapping.asDataFrame()
+    # New table name
+    table_name = 'Narrow MAF {} Database'.format(time.time())
+
+    #Get main project id
+    projectid = process_functions.getDatabaseSynId(
+        syn, "main", databaseToSynIdMappingDf=database_mappingdf)
+
+    new_tables = process_functions.create_new_fileformat_table(syn, database_mapping,
+                                                               "vcf2maf",
+                                                               table_name,
+                                                               projectid,
+                                                               "syn7208886")
+
+    # Remove can download permissions from project GENIE team
+    syn.setPermissions(new_tables['newdb_ent'].id, 3326313, [])
```

Comment:
this wouldn't be necessary if just move to a projec that isn't viewable by GENIE peeps.